ui/src/dataLoaders/components/collectorsWizard/select/SelectCollectorsStep.tsx

```diff
@@ -61,14 +61,14 @@ export class SelectCollectorsStep extends PureComponent<Props> {
           <h5 className="wizard-step--sub-title">
             Looking for other things to monitor? Check out our 200+ other &nbsp;
             <a
-              href="https://v2.docs.influxdata.com/v2.0/reference/telegraf-plugins/#input-plugins"
+              href="https://docs.influxdata.com/telegraf/v1.17/plugins/#input-plugins"
```

Comment:
Not a show-stopper, but you may update this to `https://docs.influxdata.com/telegraf/latest/plugins/#input-plugins` just so it doesn't need to be updated for each Telegraf release.

```suggestion
              href="https://docs.influxdata.com/telegraf/latest/plugins/#input-plugins"
```

Reply:
Would it make sense to do the same for all the links that start with `docs.influxdata/com/influxdb/v2.0`?

Reply:
This is where it gets tricky. The InfluxDB docs are versioned per minor version of InfluxDB, so it makes sense to leave these as-is and then update them to v2.1 whenever 2.1 rolls out. That way the links in the UI point to the specific version of InfluxDB the user is running. The tricky part is that this same code is used (afaik) in the Cloud UI, so links in cloud are actually pointing to OSS docs. So in short... I don't know 🤷🏻‍♂️ 

In an ideal world, we'd be able to have these links be version/context specific.

Reply:
What I'll do is:
* Update the links on `master` to use `latest`
* When I backport this to the `2.0` branch, keep the links pointing at `2.0`